app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1036,11 +1036,26 @@ class PatientRepositoryAndroidTest {
     assertEquals(creationTimeOf2ndBp, recentPatientAfter2ndBp.lastBp!!.createdAt)
   }
 
+  @Test
+  fun verify_deleted_bps_are_not_picked_up() {
+    val facilityUuid = testData.qaUserFacilityUuid()
+    savePatientWithBp(facilityUuid = facilityUuid)
+    savePatientWithBp(facilityUuid = facilityUuid, deletedAt = Instant.now())
```

Comment:
Please pass the `clock` in `Instant.now()`

Reply:
We introduced the `clock` at a later point in our code. The reason for introducing a `clock` was to avoid configuring time-zones in various places and it also makes testing easier.
So from now on, wherever we are using an instance of time or date, we inject the `clock`.

If you want, we can pair on this change.

Reply:
I'll work on this as a part of another story and raise a PR for that. Story: https://www.pivotaltracker.com/story/show/164599452